rules/base-WINDOWS-rules.ham

```diff
@@ -0,0 +1,31 @@
+# base.ham is setup for GCC by default
```

Comment:
This should be in toolkit-cpp-clang_33.ham. See the `pierre-linux-clang` branch (PR #10) where I've done the same thing for Linux.